templates/search_results.hbs

```diff
@@ -23,9 +23,9 @@
               <path stroke="currentColor" stroke-linecap="round" d="M3 9l6-6m0 6L3 3"/>
             </svg>
           </button>
-          <h3 class="collapsible-sidebar-title sidenav-title">{{t 'filter_by_help_center'}}</h3>
+          <h3 class="collapsible-sidebar-title sidenav-title">{{t 'filter_source'}}</h3>
```

Comment:
Nit: do we want `filter_by_source` to match the previous pattern?

Reply:
I wanted that initially, but to keep it consistent I had to choose this, since we can't have `filter_by_type` - since that's in use already and added a new `filter_type` instead :/

Reply:
Why is naming things so hard?